### ISSUE TITLE
Duplicate `containedIn` values

### ISSUE THREAD

**Issue by fsteeg** [hbz/lobid-resources#1816]
Reported by I.N. via email on 2023-07-11

see e.g. `containedIn` for http://lobid.org/resources/99372483173006441.json

Looking at https://alma.lobid.org/marcxml/99372483173006441 it seem to be coming in twice, is this related to a bridge (the second entry has `49HBZ_BRIDGE_BDR`)? Quoting from the mail: "die HT-Nr. im Feld 773 [kommt in Alma] nur einmal vor".

**Comment by TobiasNx** [comment]
Yes this seems to be an issue of a local field. I will uniq the property. This should fix this.
